src/main/resources/rules/HTL-5.md

```diff
@@ -0,0 +1,24 @@
+Always try to re-use existing conditions, so the code is more readable.
+
+== Noncompliant Code Example
+```
+<!--/* Bad - the same condition is evaluated multiple times */-->
+<span class="uber-mode__top-bar" data-sly-test="${uberModeHelper.uberModeEnabled || forceUberMode}">
+    <div class="my-component">
+        Blah blah blah
+    </div>
+</span>
+<span class="uber-mode__bottom-bar" data-sly-test="${uberModeHelper.uberModeEnabled || forceUberMode}"></span>
+```
+ 
+ 
+    
+== Compliant Solution
+```
+<span class="uber-mode__top-bar" data-sly-test.uberMode="${uberModeHelper.uberModeEnabled || forceUberMode}">
+    <div class="my-component">
+        Blah blah blah
```

Comment:
Could we be more eloquent than Blah blah blah ?

Reply:
maybe :), changed to: "Some text".